templates/terraform/examples/base_configs/test_file.go.erb

```diff
@@ -10,6 +10,8 @@ import (
   "github.com/hashicorp/terraform/helper/resource"
 )
 <%
+raise 'skip_test should be true if resource is not importable' \
```

Comment:
What about skipping the import step when generating tests instead? I'm not a fan of 'knowing' which configurations need to be used together and finding out at build time.

Reply:
Import is pretty important for the autogenerated tests since we use it to confirm the attributes. In general, I don't really anticipate exclude_import being used much (i.e. pretty much when the resource contains sensitive information that cannot be returned) and the verification of test resources behavior will probably be unique, so it may not make sense?